components/mpas-ocean/cime_config/buildnml

```diff
@@ -1314,6 +1314,7 @@ def buildnml(case, caseroot, compname):
             lines.append('    <var name="peakWaveFrequency"/>')
             lines.append('    <var name="peakWaveDirection"/>')
             lines.append('    <var name="significantWaveHeight"/>')
+            lines.append('    <var name=""langmuirNumber/>')
```

Comment:
double quote out of place

Reply:
thanks for catching that, @mark-petersen